### DISCUSSION TITLE
What are the benefits of Fresh over Astro?

### DISCUSSION THREAD

**Discussion by kawpii** [freshframework/fresh#615] [Q&A]
Other than not having build step (but is not doing transpiling JIT increase page response time?)

**Comment by neezer** [comment]
No expert here, but having recently surveyed the landscape and done my own comparison between Astro and Fresh, I think it's safe to say Astro is more focused on content-heavy websites like blogs, cms, etc. These kinds of sites can have optimization steps that work for their content that wouldn't work for other types of sites.

Astro has a great little section on their documentation website that covers this difference: https://docs.astro.build/en/concepts/why-astro/#content-focused

> Astro was designed for building content-rich websites. This includes most marketing sites, publishing sites, documentation sites, blogs, portfolios, and some ecommerce sites.
> 
> By contrast, most modern web frameworks are designed for building web applications. These frameworks work best for building more complex, application-like experiences in the browser: logged-in admin dashboards, inboxes, social networks, todo lists, and even native-like applications like [Figma](https://figma.com/) and [Ping](https://ping.gg/).
> 
> This is one of the most important differences to understand about Astro. Astro’s unique focus on content lets Astro make tradeoffs and deliver unmatched performance features that wouldn’t make sense for more application-focused web frameworks to implement.

Fresh seems more tailored to building interactive web applications. This was one reason (of many) that I ended up picking Fresh over Astro for my project.

**Comment by shishkin** [comment]
Astro supports SSG, Fresh doesn't.
